package.json

```diff
@@ -41,8 +45,9 @@
     "prettier": "prettier --write {,*/**/}*.{js,test.js,sol}",
     "lint-contracts": "solhint contracts/**/*.sol",
     "test": "brownie test",
-    "test-js": "truffle test",
-    "coverage": "truffle run coverage --file='tests-js/'"
+    "test-js": "echo Executing pure Hardhat test cases ... && npx hardhat test && echo Executing test cases using Ganache ... && npx hardhat --config hardhat-ganache-tests.config.js --network ganache test",
```

Comment:
You might have to add this: https://www.npmjs.com/package/hardhat in order for the current CI to work. The tests are failing due to hardhat package not found I believe. Could you please check that?

Reply:
done

Reply:
Could you push it here?

Reply:
done